libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -168,20 +224,20 @@ class CreatorNode {
     }
   }
 
-  getEndpoint () {
+  getEndpoint() {
     return this.creatorNodeEndpoint
   }
 
   /**
    * Switch from one creatorNodeEndpoint to another including logging out from the old node, updating the endpoint and logging into new node */
-  async setEndpoint (creatorNodeEndpoint) {
+  async setEndpoint(creatorNodeEndpoint: string) {
     // If the endpoints are the same, no-op.
     if (this.creatorNodeEndpoint === creatorNodeEndpoint) return
 
     if (this.connected) {
       try {
         await this._logoutNodeUser()
-      } catch (e) {
+      } catch (e: any) {
```

Comment:
e: Error? and e?.message perhaps later on?

Reply:
weirdly `catch (e)` can only be typed as `unknown` or `any` so doing any here, and then maybe what i should do is next line go `const error = e as Error`?